### PR TITLE
feat: command line argument for specifying output path

### PR DESCRIPTION
**Pull request by charlesbdudley**

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -32,14 +33,26 @@ const download = (url, cb) => {
...
 
+// Parse command line arguments
+const argumentDefinitions = [
+  {
+    name: 'outputPath',
+    type: String,
+  },
+]
+const args = commandLineArgs(argumentDefinitions)
+const outputPath = args.outputPath || './coverage/badge.svg'
```
Comment (tlvince) [inline cli.js]: Looks like you could use commandLineArgs's [defaultValue](https://github.com/75lb/command-line-args/blob/9c07457557351d3db825bd5a3d17c053d6a10dea/doc/option-definition.md#module_option-definition--OptionDefinition+defaultValue) instead.

**Review by tlvince — CHANGES_REQUESTED** [review]
Hey! Thanks. This looks good. Could you just fix up the [linting errors](https://travis-ci.org/tlvince/make-coverage-badge/builds/357524540#L474) and have a look at `defaultValue`? Otherwise, this lgtm.

**Comment by charlesbdudley** [comment]
@tlvince Ah, didn't understand what the "standard" meant in the test script. Fixed now.

Also, I added the `defaultOption` key in the command-line-args config to set this up for potential future command line options. Let me know if you'd rather remove the flag completely.

**Comment by shawnpetros** [comment]
hey, is this getting merged soon? I really wanted to use this and am looking for this exact functionality!

File: cli.js (hunk 1 above)
Comment (charlesbdudley) [reply cli.js]: thanks @tlvince When you get a chance can you please review the changes?

**Review by tlvince — APPROVED** [review]

**Pull request merged**